src/Resources/translations/messages.de.yml

```diff
@@ -1,2 +1,2 @@
 # labels
-huh.google_maps.label.routing: Routenplaner zu Ihrer Adresse:
\ No newline at end of file
+huh.google_maps.label.routing: 'Routenplaner zu Ihrer Adresse:'
```

Comment:
Unnecessary. Why did you add the apostrophes?

Reply:
Well the inspector of phpstorm complained that it's not best practice without quotes!

Reply:
Mhm, I guess it's a question of taste (https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml), so no problem.

Reply:
Yes you're right - I can also revert if you like!

Reply:
Yes, please.

Reply:
Okay, my revert was to late - now the PR is with the quotes!